.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+# HTCondor Pull Request Checklist for internal reviewers
```

Comment:
Leave something at the top reminding PR submitters to place a brief description? Or do we not care about this given that one should already exist in the Jira ticket?

```suggestion
<Insert PR description here, and leave checklist below for code review.>

# HTCondor Pull Request Checklist for internal reviewers
```